onnxruntime/core/graph/graph_utils.h

```diff
@@ -10,15 +10,20 @@
 #endif
 #include "onnx/onnx_pb.h"
 #include "onnx/onnx-operators_pb.h"
-#include "core/graph/graph.h"
 
 #include "core/common/inlined_containers.h"
+#include "core/graph/graph.h"
 
 namespace onnxruntime {
-
 namespace graph_utils {
 
-#if !defined(ORT_MINIMAL_BUILD) || defined(ORT_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD)
```

Comment:
moved code so that less specific sections are first.
all builds -> `!defined(ORT_MINIMAL_BUILD) || defined(ORT_EXTENDED_MINIMAL_BUILD)` -> `!defined(ORT_MINIMAL_BUILD)`